test/tic_tac_toe_test.exs

```diff
@@ -17,8 +17,14 @@ defmodule TicTacToeTest do
         """
     end
 
+    test "(integration test) displays input message" do
+      user_input = "1"
+      assert capture_io(user_input, fn -> IO.write(TicTacToe.start(CommandLine)) end) =~
+        "Please input desired placement: #{user_input}"
+    end
+
     test "(unit test) displays welcome message and board" do
-      assert TicTacToe.start(WriterMock) ===
+      assert TicTacToe.start(CommunicatorMock) =~
```

Comment:
With how the mock exists today, when either `read_input` or `display` is called via the `TicTacToe` application code under test, we'll be spawning a new unsupervised process. Maybe we could design this in a way where we explicitly spawn the process for the `CommunicatorMock` prior to calling `TicTacToe.start/1` in our test can properly shutdown the process when we're done with it.